src/pyOpenMS/pxds/MzIdentMLFile.pxd

```diff
@@ -17,6 +17,29 @@ cdef extern from "<OpenMS/FORMAT/MzIdentMLFile.h>" namespace "OpenMS":
         MzIdentMLFile() nogil except +
 
         void load(String filename, libcpp_vector[ProteinIdentification] & poid, libcpp_vector[PeptideIdentification] & peid) nogil except +
+            # wrap-doc:
+                #   Loads the identifications from a MzIdentML file
+                #   -----
```

Comment:
param?

Reply:
Didn't found details about the param in the header file.